src/utils/constants.ts

```diff
@@ -89,10 +89,11 @@ export const DEFAULT_TABLE_SETTINGS: Settings = {
     highlightRows: true,
 } as const;
 
-export const TENANT_OVERVIEW_TABLES_SETTINGS = {
+export const TENANT_OVERVIEW_TABLES_SETTINGS: Settings = {
     ...DEFAULT_TABLE_SETTINGS,
     stickyHead: 'fixed',
     dynamicRender: false,
+    sortable: false,
```

Comment:
Tenant tables display tops, there is no need in sorting there (and this helps to get rid of unneeded code that modifies columns)